config/config.go

```diff
@@ -59,6 +59,8 @@ type BlockManagerConfig struct {
 	BatchSkew uint64 `mapstructure:"max_batch_skew"`
 	// The size of the batch of blocks and commits in Bytes. We'll write every batch to the DA and the settlement layer.
 	BatchSubmitBytes uint64 `mapstructure:"batch_submit_bytes"`
+	// SequencerUpdateInterval defines the interval at which to fetch sequencer updates from the settlement layer
+	SequencerUpdateInterval time.Duration `mapstructure:"sequencer_update_interval"`
```

Comment:
lets please change to `SequencerSetUpdateInterval` vs `Sequencer..` in order to match the function name and also its the actual set update.